editor/utils/dom.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Given a block UID, returns the corresponding DOM node for the block, if
+ * exists. As much as possible, this helper should be avoided, and used only
+ * in cases where isolated behaviors need remote access to a block node.
+ *
+ * @param {string} uid Block UID.
+ *
+ * @return {Element} Block DOM node.
+ */
+export function getBlockDOMNode( uid ) {
+	return document.querySelector( '[data-block="' + uid + '"]' );
```

Comment:
I'm seeing two other places where we use a similar technical to get the block DOM node, should we update those as well? (search for `[data-block="`)

Reply:
>I'm seeing two other places where we use a similar technical to get the block DOM node, should we update those as well? (search for `[data-block="`)

I could only find one other, but replaced it in fad28e2.

There's a reference to `data-block` in `WritingFlow` as well, but this wouldn't be as easy to replace and is proposed for removal anyways in #5289.